### DISCUSSION TITLE
Update translations after app has been built

### DISCUSSION THREAD

**Discussion by yummyhotdog** [nuxt-modules/i18n#1341] [Q&A]
Hello!

I am trying to figure out how to update my i18n translations in production, after the app has been built & deployed. I tried using the `lazy` option (with and without `skipNuxtState`) without any luck. Here's my current nuxt.config.js:
```
...
i18n: {
    locales: [
      {
        code: 'en',
        iso: 'en-US',
        file: 'en-US.js'
      },
      {
        code: 'de',
        iso: 'de-DE',
        file: 'de-DE.js'
      }
    ],
    lazy: {
      skipNuxtState: true
    },
    langDir: './locales/',
    strategy: 'no_prefix',
    defaultLocale: 'en',
    vueI18n: {
      fallbackLocale: 'en'
    }
...
```
Use case: After the app has been deployed, I want to update a string inside `locales/en-US.js`. The app should update the respective string in the frontend, without the need of restarting the server or redeploying the app. Is this possible?

Any help is appreciated, thank you! :)

**Comment by rchl** [comment]
Your translation files (like `locales/en-US.js`) can return a function. In that case the module will call the function on every request to retrieve the translations. With that in place, it would then be up to you to figure out how to return updated translations one way or another. You could for example use `redis` on the server side to cache the translations (or just cache in memory if you have only one instance) and expose some endpoint for triggering an update of translations.

**Reply by yummyhotdog** [reply]
Good point, thanks for your input! The thing is, I want to avoid any call to the server from the app to retreive my translations. The files `de-DE.js` and `en-US.js` will be managed and updated externally. I just need the application to always use the actual content files, and not the content at the moment of deployment.

**Reply by rchl** [reply]
With Nuxt everything gets bundled so it's not really possible to replace individual files like that. Or at least I don't know of any clean way.
